core/models.py

```diff
@@ -41,3 +42,6 @@ class Review(models.Model):
     travel_allowance = models.IntegerField()
     pension = models.IntegerField()
     quality = models.IntegerField()
+
+    def get_absolute_url(self):
+        return reverse('review-detail', args=[str(self.pk)])
```

Comment:
I'm not sure exactly how to change it, but "allow edits from maintainers" is checked so feel free..